tests/chainer_tests/links_tests/connection_tests/test_convolution_nd.py

```diff
@@ -180,4 +180,35 @@ def test_conv3d(self):
         testing.assert_allclose(link_nd(x).data, link_3d(x).data)
 
 
+@testing.parameterize(*testing.product({
+    'ndim': [1, 3],
+    'nobias': [True, False],
+    'groups': [1, 5],
+    'device': [
+        '@numpy',
+        '@intel64',
+        '@cupy:0',
+        'native:0',
+        'cuda:0',
+    ]
+}))
+class TestConvolutionNDFromParams(unittest.TestCase):
```

Comment:
Instead of adding a new Test case, 
How about using the from_params in the `create_link` of the `testing.LinkTestCase`

Reply:
Sure.

Is this applicable to the other test cases?

Reply:
yes,
rebase with the master branch so the new tests appear :)

You can ignore this suggestion if you dont feel comfortable with this change